Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -3180,6 +3180,13 @@ void WebPageProxy::handleMouseEvent(const NativeWebMouseEvent& event)
         processNextQueuedMouseEvent();
 }
 
+void WebPageProxy::shouldFireMouseDidMoveOverElement(const NativeWebMouseEvent& event)
```

Comment:
Hm… `shouldFireMouseDidMoveOverElement` is a bit of an odd name for this function, since it reads like a method that should return a `bool` with no other side effects.

Perhaps something like `fireMouseDidMoveOverElementAfterHitTest()` or `dispatchMouseDidMoveOverElementAsynchronously()`?

Reply:
Okay yes I see how the name is misleading. I'll just rename to `dispatchMouseDidMoveOverElementAsynchronously` with the idea that the callee needn't know why a hit test is needed for this dispatch.

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/b7607875e50a6089c8262d0f679251a1a5216add